.github/modules/retry/action.yml

```diff
@@ -0,0 +1,40 @@
+# Composite function to retry a command for a maximum of 3 times
```

Comment:
Should we maybe start with 2 times?

Concerned that we could be eating up minutes.

Reply:
we're using 3 on EXIPs and it will only go 3 times if something keeps failing.  Most of the time it will retry twice and pass on the second attempt.  I think 2 should be fine too.  @abhi-markan opinion?

Reply:
> Concerned that we could be eating up minutes.

This is a concern of mine too -- especially if it blocks out runners.  I'm not sure if adding retries is the way forward? 

Sorry I'm out the loop here -- I'm assuming this retry the entire test suite, which will massively increase network calls due to existing flakeyness

Is there not a way using jest.retryTimes to only run the failed tests as apposed to the entire suite?